src/wopiserver.py

```diff
@@ -347,11 +354,18 @@ def iopOpenInApp():
                       (req.remote_addr, userid, username, viewmode, endpoint, e))
         return 'Remote error, file not found or file is a directory', http.client.NOT_FOUND
 
+    res = {}
     if bridge.issupported(appname):
-        return bridge.appopen(url_unquote(utils.generateWopiSrc(inode)), acctok)
-    return flask.redirect('%s&WOPISrc=%s&access_token=%s' %
-                            (appurl if viewmode == utils.ViewMode.READ_WRITE else appviewurl,
-                            utils.generateWopiSrc(inode), acctok))      # no need to URL-encode the JWT token
+        a, b = bridge.appopen(url_unquote(utils.generateWopiSrc(inode)), acctok)
```

Comment:
Better naming of the vars?

Reply:
Yeah this is because the response has multiple meanings, but that's not very pythonic. Will refactor the code with exception handling